src/libxively/sft/xi_sft_logic.c

```diff
@@ -7,6 +7,10 @@
 #include <xi_sft_logic.h>
 #include <stddef.h>
 #include <xi_macros.h>
+#include <xi_debug.h>
+
+// #define XI_SFT_DOWNLOAD_BYTES_PER_FILE_CHUNK 1 * 1024
+#define XI_SFT_DOWNLOAD_BYTES_PER_FILE_CHUNK 4 * 1024
```

Comment:
Also this should be in an easy to configure header somewhere so that customers can easily... customize it.